### PR TITLE
Use subclaassed plugin downloaders during the pulp repair.

### PR DESCRIPTION
**Pull request by ipanova**
closes #7909
https://pulp.plan.io/issues/7909

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

**Review by dralley — APPROVED** [review]

File: CHANGES/7909.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Use subclaassed plugin downloaders during the pulp repair.
```
Comment (bmbouter) [inline CHANGES/7909.bugfix:1]:
```suggestion
Use subclassed plugin downloaders during the pulp repair.
```

**Review by bmbouter — CHANGES_REQUESTED** [review]
nitpick change requested, then lgtm.

**Comment by ipanova** [comment]
@bmbouter merge ahead when ci passes

**Review by bmbouter — APPROVED** [review]
Thank you!

**Pull request merged**